mlflow/java/spark/src/main/scala/org/mlflow/spark/autologging/MlflowAutologEventPublisher.scala

```diff
@@ -145,11 +145,15 @@ private[autologging] trait MlflowAutologEventPublisherImpl {
     }
   }
 
+  // https://github.com/delta-io/delta/blob/aaf3cd77dae06118f5cb7716eb2e71c791c6a148/core/src/main/scala/org/apache/spark/sql/delta/util/FileNames.scala#L26
+  private val checkpointFilePattern = "\\d+\\.checkpoint(\\.\\d+\\.\\d+)?\\.parquet".r.pattern
+  private def isCheckpointFile(path: String): Boolean = checkpointFilePattern.matcher(path).matches()
+
   private[autologging] def publishEvent(
       replIdOpt: Option[String],
       sparkTableInfo: SparkTableInfo): Unit = synchronized {
     sparkTableInfo match {
-      case SparkTableInfo(path, version, format) =>
+      case SparkTableInfo(path, version, format) if !isCheckpointFile(path) =>
```

Comment:
Could you comment more context about delta checkpointing file ?

Reply:
@WeichenXu123 Added a comment

Reply:
Can we add a test for this fix ?

Reply:
@WeichenXu123 Good suggestion. Can we add a test in a follow-up PR?